content/docs/getting-started/assigning-admins/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Designating admins
+publishedDate: '2023-08-31'
+description: How to designate yourself as an admin of Roadie Backstage.
+---
+
+## Introduction
+
+You often want to allow power users to edit the layouts and install plugins while preventing
+ordinary users from making unintended changes to the application setup. Roadie Backstage has RBAC built in to enable this.
+
+⚠️  Admins have no ability to read secrets or access other secure information inside Roadie.
+
+## Setup
+
+Initially the first user will be assigned the Admin role and will be able to add other users as well as assign granular roles to them in the User Management page in Administration.
+
+![User management screen](./user-management.png)
+
+You can add roles to users in this page using the Edit pencil icon next to each user.
+
+![Editing roles for a user](./edit-user-roles.png)
+
+Roadie comes with a predefined set of Roles to use for access control to core app functionality. Custom roles can be configured via the Role Management tab using our predefined set of permissions. 
+
+![Adding roles to a user](./add-roles.png)
+
+## Non-Admins
+
+Non-admins will not be able to access any app level configuration like editing plugin layouts, configuring new plugins or defining plugin config. 
+
+You can view the permissions for the Admin group via the Role Management tab in the Administration section. 
```

Comment:
We could link to this page at the bottom here I think https://roadie.io/docs/details/permissions/